settings.gradle

```diff
@@ -8,8 +8,6 @@ pluginManagement {
 
 include ':feature:amazon'
 include ':integration-tests'
-include ':test-utils'
```

Comment:
It was easier to remove the `test-utils` now in this PR and merge it in `purchases` so I removed it